src/api/middleware/validators/offer.js

```diff
@@ -333,7 +333,7 @@ export const edit = useExpressValidators([
     body("title", ValidationReasons.DEFAULT)
         .optional()
         .isString().withMessage(ValidationReasons.STRING)
-        .isLength({ max: OfferConstants.title.max_length }).withMessage(ValidationReasons.TOO_LONG(90))
+        .isLength({ max: OfferConstants.title.max_length }).withMessage(ValidationReasons.TOO_LONG(OfferConstants.title.max_length))
         .trim(),
```

Comment:
I just noticed there's a missing validator here and in create. You need to add a min parameter to the offer title, let's set it to 1 for now. If we try to create an offer with an empty title, we will get an unexpected error.
What you gotta do is:

- Add an isLenght with `ValidationReasons.TOO_SHORT` and with `min: OfferConstants.title.min_length` validator in [here](https://github.com/NIAEFEUP/nijobs-be/blob/b3570e2561799f89543f04e5195a04b1eb7d9735/src/api/middleware/validators/offer.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L77) and [here](https://github.com/NIAEFEUP/nijobs-be/blob/b3570e2561799f89543f04e5195a04b1eb7d9735/src/api/middleware/validators/offer.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L336)
- Create an entry in offer constants
- Create a test that verifies this behavour